Uptime.CredentialManager.Web/Controllers/CredentialsController.cs

```diff
@@ -312,15 +312,19 @@ private async Task<IEnumerable<Credential>> SearchAsync(string term)
         // GET: Credentials/Search
         public IActionResult Search()
         {
-            var credential = new List<Credential>();                    
+            var credential = new List<Credential>();
+            var identity = User.Identity as ClaimsIdentity;
```

Comment:
I think you can see the User value straight in the view, no need to pass it via ViewBag